DevOps/frankfurter/scripts/tools/sendlogs.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+HOME=ubuntu
+USER=192.168.7.2
+
+#format: ssh ubuntu 192.168.7.2 "Linux commands"
+#makes a temp directory and goes into the temp directory once made
+#gets the logs from the services in the beaglebone and logs in temp directory
+#zips log files and prints the zip file name if it has been created
+#exit ssh
+
+ssh $HOME@$USER "
+mkdir -p /tmp/log-fetch
+cd /tmp/log-fetch
+
+today=$(date +%Y-%m-%d.%H:%M:%S)
+
+journalctl -e -u update.service > update.log"."$today
+journalctl -e -u runtime.service > service.log"."$today
+journalctl -e -u networking.service > networking.log"."$today
+
+cd 
+
+tar -cjvf archivelogs.tar.bz2 tmp 
+
```

Comment:
you cd back to home, and then try to tar a file tmp within home, which doesn't exist. Should be /tmp/log-fetch (just /tmp would tar the entire /tmp folder)

Reply:
actually, after you tar the archivelogs.tar file, it outputs the logs within the folder log-fetch, which is within the folder tmp if bzip is run on /tmp/log-fetch. But it would be more helpful if only the log-fetch folder was outputted rather than having a tmp folder be outputted, with the log-fetch folder within that tmp folder.